src/features/movieDetails/index.js

```diff
@@ -0,0 +1,86 @@
+import { Navigation } from "../../common/Navigation"
+
+
+export const MovieDetailsPage = () => ({
+
+    title = "Mulan",
+    year = "2020",
+    category = ["action", "fantasy", " thiller"],
+    rate = 7.8,
+    vote = 3876,
+    place = "China, United States of America",
+    date = "24.10.2020",
+    namePerson = "Liu Yifei",
+
+}) => {
+    return (
+        <Wrapper>
+            <Navigation />
+            <PosterContainer>
+                <Poster />
+                <Title />
+                <RatingPoster>
+                    <StarPoster />
+                    <RatePoster>{rate.toFixed(1)}</RatePoster>
+                    <VotePoster>
+                        {vote} {vote > 1 ? "votes" : "vote"}
+                    </VotePoster>
+                </RatingPoster>
+            </PosterContainer>
+            <Content>
+                <MovieTile>
+                    <Poster />
+                    <MovieData>
+                        <MovieTitle>{title}</MovieTitle>
+                        <MovieYear>{year}</MovieYear>
+                        <ProductionData>
+                            <ProductionPlace>
+                                <Name>Production</Name>
+                                <Place>{place}</Place>
+                            </ProductionPlace>
+                            <ReleaseData>
+                                <Name>ReleaseData</Name>
+                                <Date>{date}</Date>
+                            </ReleaseData>
+                        </ProductionData>
+                        <CategoryTagWrapper>
+                            <CategoryTag>{category}</CategoryTag>
+                        </CategoryTagWrapper>
+                        <RatingWrapper>
+                            <Star />
+                            <RateContainer>{rate.toFixed(1)}</RateContainer>
+                            <VoteContainer>
+                                {vote} {vote > 1 ? "votes" : "vote"}
+                            </VoteContainer>
+                        </RatingWrapper>
+                        <Description>
+                            A young Chinese maiden disguises herself as a male warrior in order to save her father.
+                            Disguises herself as a male warrior in order to save her father.  A young Chinese maiden
+                            disguises herself as a male warrior in order to save her father.
+                        </Description>
+                        <Cast>
+                            <Header />
+                            <CastContent>
```

Comment:
What if There are 10 Tiles? How is this CastContent styled to take mare than 1 Tile !?